python/servo/package_commands.py

```diff
@@ -211,10 +211,9 @@ def package(self, release=False, dev=False, android=None, debug=False, debugger=
 
             print("Creating dmg")
             os.symlink('/Applications', dir_to_dmg + '/Applications')
-            dmg_path = '/'.join(dir_to_build.split('/')[:-1]) + '/'
             time = now.replace(microsecond=0).isoformat()
             time = time.replace(':', '-')
-            dmg_path += time + "-servo-tech-demo.dmg"
+            dmg_path = path.join(dir_to_build, time + "-servo-tech-demo.dmg")
```

Comment:
If it fits the line limit, I'd prefer string formatting for the second arg:
`'{}-servo-tech-demo.dmg'.format(time)`